deptofdefense/move.mil
# Batch seeding

## Checklist

I have…


## Summary of Changes

This pull request uses activerecord-import to load 500 zip code tabulation areas at a time into the database during seeding, instead of 1 at a time. Seeding goes from taking multiple minutes to just a few seconds with this change.

## Testing

To verify the changes proposed in this pull request…

1. clone this repo,
1. git checkout `batch-seeding`,
1. set up development dependencies according to [CONTRIBUTING.md](https://github.com/deptofdefense/move.mil/blob/master/CONTRIBUTING.md),
1. run `bin/rails server`, and
1. load up [http://localhost:3000](http://localhost:3000) in your Web browser of choice.
